raiwidgets/setup.py

```diff
@@ -27,7 +27,9 @@
     author="Roman Lutz, Ilya Matiach, Ke Xu",
     author_email="raiwidgets-maintain@microsoft.com",
     description="Interactive visualizations to assess fairness, explain "
-                "models, and analyze errors in Machine Learning",
+                "models, generate counterfactual examples, analyse "
```

Comment:
nit: analyse or analyze?  analyze is the US spelling and analyse is the British spelling.  It seems we have it in both ways in the same sentence here.

Reply:
Exactly, I have both to make sure we represent most people in the world who speak/write English :)

Would change this to be US spelling.